packages/vm/src/eei/vmState.ts

```diff
@@ -266,6 +266,7 @@ export class VmState implements EVMStateAccess {
       }
     }
     await this._stateManager.flush()
+    this.touchedJournal.clear()
```

Comment:
I think we need to investigate this a bit further. The problem with the client test failing (`getTransactionReceipt.spec.ts`) was that:

- The blockchain is initialized with a Geth genesis state
- This creates accounts for a big precompile range (0x00 - 0x00..00ff)`
- However, when creating genesis (in `vmexecution.open()`), these accounts are all created and dumped into `touchedJournal`. This is /not/ cleared
- However, on current master, the touched accounts are empty

This sounds a bit error prone to me and I want to investigate why this change has happened.